### PR TITLE
Added evidently examples

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description

Added examples for production monitoring with Evidently

__Related to:__ `Evidently support example`

__Any expected test failures?__
Tests on python 3.11 might fail

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [X] adds a new feature
- [ ] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [X] Refactored code ([sourcery](https://sourcery.ai/))
- [X] Tested code locally
- [X] Precommit installed and run before pushing changes
- [X] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [X] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS: Windows11
- Python version: 3.8.15
- Neptune version: 1.1.1
- Affected libraries with version: evidently==0.3.1

### CODE REVIEW

File: integrations-and-supported-tools/evidently/scripts/Neptune_Evidently_reports.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+run["data_stability/report"].upload("data_stability.html")
+run["data_drift/report"].upload("data_drift_report.html")
+
+# (Neptune) Save reports as dict
+
+run["data_stability"] = stringify_unsupported(data_stability.as_dict())
+run["data_drift"] = stringify_unsupported(data_drift_report.as_dict())
+
+# (Neptune) Stop logging (optional for scripts)
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/scripts/Neptune_Evidently_reports.py:55]: IMO we can just `stop()` and leave it at that

```suggestion
# (Neptune) Stop logging
```

File: integrations-and-supported-tools/evidently/scripts/Neptune_Evidently_drifts.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            bike_df.loc[date[0] : date[1]],
+            column_mapping=data_columns,
+        )
+
+        for feature in metrics:
+            run["drift"][feature[0]].append(
+                round(feature[1], 3),
+                timestamp=datetime.strptime(date[0], "%Y-%m-%d %H:%M:%S").timestamp(),
+            )
+            # Passing a timestamp in the append methods lets you visualize the date in the x-axis of the charts
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/scripts/Neptune_Evidently_drifts.py:108]:
```suggestion
            # Passing a timestamp in the append() method lets you visualize the date in the x-axis of the charts
```

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log reports\n",
+    "\n",
+    "This section shows how you can log Evidently test suites and reports to Neptune.  \n",
+    "You can find the entire list of Evidently pretests [here](https://docs.evidentlyai.com/presets/all-presets)."
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:117]: Make link text more descriptive

```suggestion
    "You can find the entire list of pretests in the [Evidently documentation](https://docs.evidentlyai.com/presets/all-presets)."
```

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 4]
```diff
@@ -0,0 +1,565 @@
...
+    "Explore the run (reports, dictionaries) in the Neptune app, or check this [example dashboard](https://app.neptune.ai/o/common/org/evidently-support/runs/details?viewId=standard-view&detailsTab=dashboard&dashboardId=9917f940-757a-424d-879e-7781a00bf0c3&shortId=EV-7&type=run)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log production data drift\n",
+    "This section shows how you can use Evidently to evaluate production data drift and log the results to Neptune"
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:329]:
```suggestion
    "This section shows how you can use Evidently to evaluate production data drift and log the results to Neptune."
```

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 5]
```diff
@@ -0,0 +1,565 @@
...
+    "]\n",
+    "bike_df"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For demonstration purposes, we treat this data as the input data for a live model. To use with production models, the prediction logs should be available."
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:378]: > the prediction logs should be available

Maybe replace "should" with an unambiguous verb. If the logs have to be available &rarr; "must"

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 6]
```diff
@@ -0,0 +1,565 @@
...
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "HIL4YStSelut",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Analyze logged drifts in the Neptune app\n",
+    "Go to the run link and explore the drifts in the \"Charts\" dashboard. You might have to change the x-axis from \"step\" to \"Time (absolute)\".  \n",
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:527]: Here we can use bolding for the UI lables

```suggestion
    "Go to the run link and explore the drifts in the **Charts** dashboard. You might have to change the x-axis from **Step** to **Time (absolute)**.\n",
```

**Review by normandy7 — CHANGES_REQUESTED** [review]
Just some minor comments, looks good otherwise.

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb (hunk 5 above)
Comment (SiddhantSadangi) [reply integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:378]: This is copied verbatim from Evidently's documentation as this is concerned with how Evidently works and not within Neptune's scope

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 7]
```diff
@@ -0,0 +1,565 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, you can log the example to your own workspace.\n",
+    "\n",
+    "To do that, replace the code above with the following:\n",
+    "\n",
+    "```python\n",
+    "from getpass import getpass\n",
+    "\n",
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:221]: This was missing from the Notebook template, sorry about that.

```suggestion
    "from getpass import getpass\n",
    "import neptune\n",
    "\n",
```

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 8]
```diff
@@ -0,0 +1,565 @@
...
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "For example, if your workspace name is `ml-team` and the project name is `classification`, the project argument is: `project=\"ml-team/classification\"`.\n",
+    "\n",
+    "To find your API token and project name, [log in to Neptune](https://app.neptune.ai/).\n",
+    "- In the top-right corner, click your avatar and select **Get your API token**.\n",
+    "- To find and copy your project name, navigate to the project, then click **Settings** → **Properties**.\n",
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:232]: Make sure to update the UI navigation instructions if you're copying from somewhere other than the template.

```suggestion
    "- In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
    "- To copy the project name, in the top-right corner, open the settings menu and select **Properties**.\n",
```

**Review by normandy7 — CHANGES_REQUESTED** [review]

File: integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb [hunk 9]
```diff
@@ -0,0 +1,566 @@
...
+    "id": "zY6B8JTKeluQ",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "[Evidently](https://www.evidentlyai.com/) is an open-source tool to evualuate, test, and monitor machine learning models.  \n",
```
Comment (normandy7) [inline integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb:26]: Typo.

```suggestion
    "[Evidently](https://www.evidentlyai.com/) is an open-source tool to evaluate, test, and monitor machine learning models.\n",
```

**Review by normandy7 — APPROVED** [review]

**Pull request merged**